### PR TITLE
Fixing default switch field styles

### PR DESCRIPTION
**Pull request by dbanksdesign**
*Issue #, if available:*

*Description of changes:*

* CSS and design token names were not matching, causing the default background color to not show up. 
* Updated a few of the design token values for the switch field to be more semantic so they will work better in different themes/modes
* Removed default style prop values in the demo so the demo looks like the default switch field by default. 

https://user-images.githubusercontent.com/321279/142954543-808a6dfd-0760-4a13-9904-059488ac3700.mp4

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by reesscot** [comment]
Please add a changeset

**Review by ericclemmons — APPROVED** [review]

File: docs/src/pages/components/switchfield/demo.tsx [hunk 1]
```diff
@@ -7,9 +7,9 @@ import { Example } from '@/components/Example';
 
 export const SwitchDemo = () => {
   const switchProps = useSwitchProps({
-    thumbColor: '#fff',
-    trackColor: 'hsl(210, 5%, 80%)',
-    trackCheckedColor: 'hsl(220, 70%, 70%)',
+    // thumbColor: '#fff',
+    // trackColor: 'hsl(210, 5%, 80%)',
+    // trackCheckedColor: 'hsl(220, 70%, 70%)',
```
Comment (zchenwei) [inline docs/src/pages/components/switchfield/demo.tsx:12]: Remove them?

File: packages/ui/src/theme/css/component/switchField.scss [hunk 2]
```diff
@@ -36,9 +36,9 @@
   transition-duration: var(
     --amplify-components-switchfield-track-transition-duration
   );
-  background: var(--amplify-components-switchfield-track-background);
+  background: var(--amplify-components-switchfield-track-background-color);
```
Comment (zchenwei) [inline packages/ui/src/theme/css/component/switchField.scss:39]:
```suggestion
  background-color: var(--amplify-components-switchfield-track-background-color);
```

File: packages/ui/src/theme/css/component/switchField.scss [hunk 3]
```diff
@@ -36,9 +36,9 @@
...
   &[data-checked='true'] {
-    background: var(--amplify-components-switchfield-track-checked-background);
+    background: var(--amplify-components-switchfield-track-checked-background-color);
```
Comment (zchenwei) [inline packages/ui/src/theme/css/component/switchField.scss:41]:
```suggestion
    background-color: var(--amplify-components-switchfield-track-checked-background-color);
```

File: packages/ui/src/theme/css/component/switchField.scss [hunk 4]
```diff
@@ -50,7 +50,7 @@
 }
 
 .amplify-switch-thumb {
-  background: var(--amplify-components-switchfield-thumb-background);
+  background: var(--amplify-components-switchfield-thumb-background-color);
```
Comment (zchenwei) [inline packages/ui/src/theme/css/component/switchField.scss:53]:
```suggestion
  background-color: var(--amplify-components-switchfield-thumb-background-color);
```

**Review by zchenwei — APPROVED** [review]
LGTM! A few minor comments

**Comment by dbanksdesign** [comment]
Thanks @zchenwei for the suggestions! I kinda rushed through these changes, glad you caught them!

**Pull request merged**